frontend/public/components/factory/Table/VirtualizedTable.tsx

```diff
@@ -181,6 +183,29 @@ const VirtualizedTable: VirtualizedTableFC = ({
     </WindowScroller>
   );
 
+  const formatToCsv = (array, delimiter = ',') =>
```

Comment:
I think we don't need to map again and convert this to CSV data, do you think the data should be provided with the format already?